src/sonic-host-services/scripts/caclmgrd

```diff
@@ -201,6 +206,12 @@ class ControlPlaneAclManager(daemon_base.DaemonBase):
         tcp_flags_str = tcp_flags_str[:-1]
         return tcp_flags_str
 
+    def update_feature_present(self):
```

Comment:
>update_feature_present [](http://example.com/codeflow?start=8&length=22)

Could you add unit tests to cover new code?

Reply:
Added unit-test and mock data for the unit-test.